assets/js/src/segments/list.jsx

```diff
@@ -6,6 +6,10 @@ import PropTypes from 'prop-types';
 
 import Listing from 'listing/listing.jsx';
 
+const isWPUsersSegment = segment => segment.type === 'wp_users';
+const isWooCommerceCustomersSegment = segment => segment.type === 'woocommerce_users';
+const isSpecialSegment = segmt => isWPUsersSegment(segmt) || isWooCommerceCustomersSegment(segmt);
```

Comment:
I've left renaming of old "WP users" terminology to another task to make the PR smaller.

The `segmt` var was introduced because of an ESLint rule of max line length so I shortened `segment` otherwise I would need to rewrite this to a non-arrow function.

Reply:
Makes sense, thanks. Anyway, I think we should drop that lint. Or change it to some more reasonable length.

(Btw. I added a single comment by mistake, wanted to "Start a review", still working on it.)